### PR TITLE
Add rpkamp-go

### PR DESCRIPTION
**Pull request by rpkamp**
Add a raffler in go that uses goroutines and a lot of randomness.

Goroutines, Goroutines everywhere

### CODE REVIEW

**Comment by lucasvanlierop** [comment]
Cool!, would you mind add some docker config for it and adding it to:
* https://github.com/domcode/rafflers/blob/master/build.sh
* https://github.com/domcode/rafflers/blob/master/raffle.sh

Check one of the existing rafflers for inspiration, or ping me directly.

**Comment by rpkamp** [comment]
No problem, I'll add it tonight :)

**Comment by lucasvanlierop** [comment]
Thanks!

**Comment by rpkamp** [comment]
Updated.

I've also changed `build.sh` so it will automatically detect what to build, and build it - skipping containers that were already build.
I've also changed `raffle.sh` so it will automatically detect from docker what rafflers are available, so there is no more need to hardcode them.

File: build.sh [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
-set -e
+#!/bin/bash
 
-docker build -t aochagavia_rust_raffler aochagavia-rust
-docker build -t shawnmccool_scala_raffler shawnmccool-scala
-docker build -t basbl_zsh_raffler basbl-zsh
-docker build -t lucasvanlierop_cobol_raffler lucasvanlierop-cobol
-docker build -t rjkip_elixir_raffler rjkip-elixir
-docker build -t sgoettschkes_haskell_raffler sgoettschkes-haskell
+dockerfiles=$(ls */Dockerfile)
```
Comment (lucasvanlierop) [inline build.sh]: The current setup was just to get thing rolling, I was planning to improve it but thanks for fixing it!

File: raffle.sh [hunk 2]
```diff
@@ -11,11 +16,11 @@ NAMES_FILE=`realpath $WORKING_DIR/$1`
 cp $NAMES_FILE $NAMES_DIR/current
 
 # Raffle a raffler ;-)
-declare -a RAFFLER_NAMES=('basbl_zsh' 'lucasvanlierop_cobol' 'rjkip_elixir' 'sgoettschkes_haskell' 'shawnmccool_scala' 'aochagavia_rust')
+declare RAFFLER_NAMES=(`docker images | awk '{print $1}' | grep '_raffler$'`)
```
Comment (lucasvanlierop) [inline raffle.sh]: Nice!

**Comment by lucasvanlierop** [comment]
If you can just fix the merge issues then

**Comment by rpkamp** [comment]
Fixed merge conflicts and checked (and fixed) scripts using shellcheck.

merge

**Comment by lucasvanlierop** [comment]
Jeeej!

**Pull request merged**